<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="labelProfile">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbUserName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRmbPW">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoLogin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFindPW">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSignUp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <zorder>lePassword</zorder>
  <zorder>leUserName</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>labelProfile</zorder>
  <zorder>cbUserName</zorder>
  <zorder>btnLogin</zorder>
  <zorder>checkBoxRmbPW</zorder>
  <zorder>checkBoxAutoLogin</zorder>
  <zorder>labelFindPW</zorder>
  <zorder>labelSignUp</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
